homework/index.js

```diff
@@ -27,21 +27,163 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
+  function renderRepositoryInfo(repository, leftContainer) {
+    const table = createAndAppend('table', leftContainer);
+    const tbody = createAndAppend('tbody', table);
+    const firstRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', firstRow, {
+      class: 'label',
+      text: 'Repository : ',
+    });
+    const td2 = createAndAppend('td', firstRow);
+    createAndAppend('a', td2, {
+      text: repository.name,
+      href: repository.html_url,
```

Comment:
You could add another property: `target: '_blank'` to open the link in a new browser tab.